presto-main/src/test/java/com/facebook/presto/type/TestIntervalDayTime.java

```diff
@@ -116,7 +117,6 @@ public void testSubtract()
     {
         assertFunction("INTERVAL '6' SECOND - INTERVAL '3' SECOND", INTERVAL_DAY_TIME, new SqlIntervalDayTime(3 * 1000));
         assertFunction("INTERVAL '9' DAY - INTERVAL '6' DAY", INTERVAL_DAY_TIME, new SqlIntervalDayTime(3 * 24 * 60 * 60 * 1000));
-        assertFunction("INTERVAL '3' SECOND - INTERVAL '6' DAY", INTERVAL_DAY_TIME, new SqlIntervalDayTime((3 * 1000) - (6 * 24 * 60 * 60 * 1000)));
```

Comment:
why did we remove this?

Reply:
thanks for catching this. i guess i accidentally removed this test instead of the addition one that i was testing with.